middlewares/error_pages_test.go

```diff
@@ -19,13 +20,15 @@ func TestErrorPage(t *testing.T) {
 	defer ts.Close()
 
 	testErrorPage := &types.ErrorPage{Backend: "error", Query: "/test", Status: []string{"500-501", "503-599"}}
+
 	testHandler, err := NewErrorPagesHandler(*testErrorPage, ts.URL)
+	require.NoError(t, err, "Should be no error")
```

Comment:
Side note: I don't think the extra message is really needed here and with the other (changed) `NoError` calls.